toolchain-msvc.cmake

```diff
@@ -53,13 +53,14 @@ endif()
 set(CMAKE_MC_COMPILER mc)
 set(CMAKE_RC_COMPILER rc)
 if(ARCH STREQUAL "amd64")
-    set(CMAKE_ASM_COMPILER ml64)
+    set(CMAKE_ASM_MASM_COMPILER ml64)
 elseif(ARCH STREQUAL "arm")
-    set(CMAKE_ASM_COMPILER armasm)
+    set(CMAKE_ASM_MASM_COMPILER armasm)
 elseif(ARCH STREQUAL "arm64")
-    set(CMAKE_ASM_COMPILER armasm64)
+    set(CMAKE_ASM_MASM_COMPILER armasm64)
 else()
-    set(CMAKE_ASM_COMPILER ml)
+    set(CMAKE_ASM_MASM_COMPILER ml)
+    set(CMAKE_ASM_MASM_FLAGS_INIT "/Cp")
```

Comment:
Doesn't ML64 also need this compile define?

Reply:
Sure. Done.